text/stacks/0004-jammy-jellyfish.md

```diff
@@ -0,0 +1,132 @@
+# Stacks based on Ubuntu 2022.04: Jammy Jellyfish
+
+## Summary
+
+A set of stacks based on the Ubuntu 2022.04 LTS (Jammy Jellyfish) release base
+image should be developed, released, and maintained by the Stacks subteam. Like
+the existing Bionic stacks, these new stacks should come in "full", "base", and
+"tiny" variants with similar, if not identical, sets of packages pre-installed.
+
+## Motivation
+
+Ubuntu provides a long-term-support (LTS) release every 2 years in April. These
+releases are supported by Canonical for 5 years. The current Bionic base image,
+first released in April 2018, will therefore be going out of support at the end
+of March 2023. In order to ensure that Paketo continues to provide a supported
+Ubuntu-based stack for our users, we should produce a new set of stack images,
+based on Jammy, alongside the current Bionic stacks.
+
+## Detailed Explanation
+
+### Variants
+
+The Jammy stack will be delivered in 3 variants that align to those already
+offered in the Bionic stack. The variants will be called `base`, `full`, and
+`tiny` just as they are in the Bionic stack. Each should be developed to offer
+roughly the same set of OS-level package support as their Bionic equivalent.
+
+### Stack IDs
+
+Stack IDs will be given to each variant of the stack as follows:
+
+* Base: `io.buildpacks.stacks.jammy`
+* Full: `io.buildpacks.stacks.jammy`
+* Tiny: `io.buildpacks.stacks.jammy.tiny`
+
+### User IDs
+
+These stacks will differ from Bionic with regards to their UID definitions.
+Each variant will ensure that the UID for the `build` phase is different than
+the `run` phase. This change will align more closely with the
+[recommendations](https://github.com/buildpacks/rfcs/blob/main/text/0085-run-uid.md)
+outlined in the Buildpacks Specification.
+
+### "Intermediate" Images
+
+The current Bionic stack is also shipped alongside an "intermediate" or "base"
+image that included all the packages, but is not decorated with the extra
+CNB-specific metadata. The thought going back was that this type of image would
+allow third-parties to more easily build a stack as they could simply extend
+upon the "intermediate" image.
+
+In order to simplify the stack offering going forward, we will no longer ship
+this "intermediate" image. Users wishing to extend the stack can simply build
+on top of the singular stack image as they would for any other type of
+container image workflow.
+
+### Image Naming and Tagging
+
+The stacks will name and tag their release images with the following pattern:
+
+```
+paketobuildpacks/jammy-{phase}-{variant}:{version}
```

Comment:
```suggestion
paketobuildpacks/{phase}-jammy-{variant}:{version}
```
Nit: Happy to go with this per-repo approach but what do you think about moving the phase first in the order so that e..g all run image repos begin with`paketobuildpacks/run-`? My approval stand regardless, just a suggestion

Reply:
Switched to your proposed naming scheme.